core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -604,5 +623,44 @@ public static void validate() {
     checkTieredLocality();
   }
 
+  /**
+   * @return a list of worker-related configurations
+   */
+  public static List<ConfigProperty> getWorkerConfiguration() {
```

Comment:
could we generate this by iterating over all property keys and taking the ones with scope worker? java 8 stream API should let us do this in a single statement

Reply:
We do not want to check Scope = Server? like alluxio.underfs.address?

Reply:
`key.getScope().contains(Scope.WORKER)`